README.md

```diff
@@ -6,14 +6,12 @@
 
 <p align=center>
 <a href="https://pypi.python.org/pypi"><img src="https://img.shields.io/pypi/v/opsdroid.svg" alt="Current version of pypi" /></a>
-<a href="https://travis-ci.org/opsdroid/opsdroid"><img src="https://img.shields.io/travis/opsdroid/opsdroid/master.svg?logo=travis" alt="Build Status" /></a>
-<a href="https://ci.appveyor.com/project/opsdroid/opsdroid/branch/master"><img src="https://img.shields.io/appveyor/ci/opsdroid/opsdroid/master.svg?logo=appveyor" alt="Build status" /></a>
+<a href="https://github.com/opsdroid/opsdroid/actions"><img src="https://github.com/opsdroid/opsdroid/workflows/CI/badge.svg?event=push" alt="Github CI Status"></img></a>
```

Comment:
I think we want to specifically have this showing the master branch.

Reply:
I would still like this to explicitly be set to master. 

```
![CI](https://github.com/opsdroid/opsdroid/workflows/CI/badge.svg?branch=master&event=push)
```

Reply:
And the link should go here https://github.com/opsdroid/opsdroid/actions?query=workflow%3ACI